recipe/meta.yaml

```diff
@@ -34,6 +36,7 @@ requirements:
     - ruamel.yaml >=0.11.14,<0.18
     - tqdm >=4
   run:
+    - __unix                       # [unix]
```

Comment:
Why was this added as a run dependency? Seems a little weird to me.

Reply:
Since it will land on `noarch`, we need to exclude it from the index in Windows. This requirement makes the solver ignore it because `__unix` is not present on Windows.

See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies

Reply:
Remind me why the same noarch build can't also be used on Windows?

Reply:
there are windows only run dependencies... so we can do a windows and unix noarch build... the windows noarch build failed during testing with conda.exe not found error... so this PR currently produces a unix noarch build and regular windows builds...

Reply:
We can have a Windows-noarch buils and save the per-Python builds.

Reply:
@dbast What do you think about @jaimergp's suggestion of a Windows specific noarch build to cover for the Windows runtime dependencies?

Reply:
This needs to be refactored now that we've added universal menuinst support (which should remove the need for a special windows package).